content/en/apps/reference/tasks.md

```diff
@@ -223,6 +223,55 @@ module.exports = {
 };
 ```
 
+## Default resolvedIf method
+If you skip defining `resolvedIf` section in the task definition, it will default to defaultResolvedIf method which is equivalent to this:
```

Comment:
Maybe explaining the why, that if actions type is `report` and `resolvedIf`isn't defined then `defaultResolvedIf` will be executed in its place. 

Also that `defaultResolvedIf` is available in the task scope to be used and how to use it (code sample).

You can also explain with few words what `defaultResolvedIf` validates (conditions|rules) so it's clear the outcome, no need of pasting it's code since it can change slightly and be out of date easy, better that if someone is interested of looking at the implementation then they can go see the open source. 
Specially because `defaultResolvedIf` won't be modified from outside but just used.